views/shared/profiles/achievements.jade

```diff
@@ -11,6 +11,12 @@
 // <div class='achievement achievement-shield'></div>
 // <div class='achievement achievement-karaoke'></div>
 
+mixin aLink(url, label)
```

Comment:
Sure, I'll try if we can just include shared mixins^^ (maybe they are already added to mobile)